### DISCUSSION TITLE
Console CLI support:  Command Line Interface for Win, Linux, & Mac scripts

### DISCUSSION THREAD

**Discussion by Europia79** [btimofeev/UniPatcher#75] [Ideas]
I need to create & apply thousands upon thousands of patches via script for Software Preservation.

Can you add JVM Console support with a CLI (Command Line Interface) that accepts command line arguments ?

That would be extremely helpful, TY !!!

@btimofeev  quote https://github.com/btimofeev/UniPatcher/discussions/37

> I once made a port of UniPatcher to a Linux console application for myself. But I haven't published it and it doesn't match the current sources for a long time.

**Comment by btimofeev** [comment]
Hello. 

Now I don't have time to make such an application. It is better to take some other desktop application for patching. I think there are console applications for most patch formats.
